ee/clickhouse/sql/person.py

```diff
@@ -56,9 +80,101 @@
     table_name=PERSONS_TABLE
 )
 
+PERSONS_UP_TO_DATE_MATERIALIZED_VIEW = """
+CREATE MATERIALIZED VIEW persons_up_to_date
+ENGINE = AggregatingMergeTree() ORDER BY (
+    id,
+    team_id,
+    updated_at
+)
```

Comment:
Ah just realized as I was migrating this:
You want the sort order to be setup so that you can find groups easily, usually with broad to fine granularity.

For instance on this `id, team_id, updated_at` everything will be first sorted by id which will make the other order keys pretty much useless since id's will be unique.

The best order really depends on how you are going to query the table. In this case I would think that maybe `team_id, id` might be the best ordering if we are going to be picking persons out of the db by team and id. If we are looking to count the number of persons by period then we will want to do `team_id, updated_at, id` so that you can say for team x I want to look at people between a and b times.

Reply:
since this is aggregatingMT we have to have all non agg fields in the sort so the last one is prob best.